monkey/monkey_island/cc/ui/src/components/pages/RegisterPage.js

```diff
@@ -65,7 +65,7 @@ class RegisterPageComponent extends React.Component {
             <Row>
               <Col lg={8} md={8} sm={8}>
                 <h1 className='reg-title'>First time?</h1>
```

Comment:
Maybe we should change this too. WDYT?

Reply:
+1

Reply:
Or change the title to "Sign Up", and leave the button's text as "Let's go!"